kenware/Assessment
# # add assessment name object

#### What does this PR do?
- create assessment name endpoints

#### Description
- add all endpoints for assessment name
- write test for all endpoints
- rename some model fields

#### How should this be manually tested?
- Create a local copy `git fetch origin ft-add-assessment-object`
- Checkout to the branch `git checkout fft-add-assessment-object`
- Start the application with the command: `python manage.py runserver`
- On postman make a GET request to `{{base-url}}/api/v1/assessments/`
- An authenticated user should be able to carry out all crude operation on user endpoints
- To run tests use `python manage.py`
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#162187176](https://www.pivotaltracker.com/story/show/162187176)
